jib-maven-plugin/README.md

```diff
@@ -556,6 +557,29 @@ To use a different Servlet engine base image, you can customize `<container><app
 </configuration>
 ```
 
+### Skaffold Integration
+
+Jib is an included builder in [Skaffold](https://github.com/GoogleContainerTools/skaffold). Jib passes on build information to skaffold through special internal goals so that skaffold understands when it should rebuild or synchronize files. For complex builds, the defaults may not be sufficient, so the jib plugin provides a `skaffold` configuration object which exposes:
```

Comment:
```suggestion
Jib is an included builder in [Skaffold](https://github.com/GoogleContainerTools/skaffold). Jib passes build information to skaffold through special internal goals so that skaffold understands when it should rebuild or synchronize files. For complex builds, the defaults may not be sufficient, so the jib plugin provides a `skaffold` configuration object which exposes:
```